### ISSUE TITLE
Nested properties

### ISSUE THREAD

**Issue by NicolasParada** [postcss/sugarss#28]
In Sass you have [nested properties](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_properties) so you can write:
```sass
body
  font:
    family: sans-serif
    size: 16px
    weight: 300
```
and get:
```css
body {
  font-family: sans-serif;
  font-size: 16px;
  font-weight: 300;
}
```
I miss this on sugarss.

**Comment by ai** [comment]
Short answer: https://github.com/jedmao/postcss-nested-props

Full answer: SugarSS is just a alternate way to write a CSS. All features like nested rules or nested properties should be implemented in plugins. So we will have all benefits of modularity.

**Comment by NicolasParada** [comment]
Nice. Thank for you help @ai :)

**Comment by NicolasParada** [comment]
I'm having other problem. After I added `postcss-nested-props`, it only parses the first nested property. I mean:

```sass
body
  font:
    family: sans-serif
    size: 16px
    weight: 300
```
becomes:
```css
body {
  font-family: sans-serif;
  size: 16px;
  weight: 300;
}
```

**Comment by ai** [comment]
Could you disable this plugin and show me input/ouput?

**Comment by ai** [comment]
But I will look tomorrow. Nite it is time to sleep 🙈

**Comment by NicolasParada** [comment]
Following the same example. it turns into:

```css
body {
  font {
    family: sans-serif;
    size: 16px;
    weight: 300 {}
  }
}
```

**Comment by NicolasParada** [comment]
A have to say, if i just use a single file with:
```sass
.test
  margin:
    top: 0
    right: 0
    bottom: 0
    left: 0
  font:
    family: 'sans-serif'
    size: 16px
    weight: 300
```
It works as spected, but in my other file, it fails. At the top of the file I have some rules in `:root` and some in `::before` and `::after`. Maybe some of them is causing this problem. But they are rules that worked fine in Sass.

**Comment by NicolasParada** [comment]
Another case:
```sass
.test
  font:
    family: sans-serif
    size: 16px
    weight: 300
  margin:
    top: 0
    right: 0
    bottom: 0
    left: 0
```
Fails on font-weight.

**Comment by ai** [comment]
Sorry, I miss what example fails. Could you find minimum file example to reproduce issue and show me input and output.

**Comment by ai** [comment]
Fixed 4e4aca1

**Comment by ai** [comment]
Also ee97d55

**Comment by ai** [comment]
Released in 0.1.4

**Comment by NicolasParada** [comment]
Thanks @ai good work ;)
All working fine now.
